client/components/Assignments/SelectDeskTemplate.jsx

```diff
@@ -91,3 +119,5 @@ SelectDeskTemplate.propTypes = {
         privateTemplates: PropTypes.array,
     }),
 };
+
+export const SelectDeskTemplate = connect(null, null)(SelectDeskTemplateComponent);
```

Comment:
Using `connect` here should not be required (as `null, null` is being passed through).
You should be able to use `export class SelectDeskTemplate` instead